devops/arangodb.yml

```diff
@@ -50,15 +50,14 @@
       copy:
         src: '{{ ssl_cert_path }}'
         dest: /home/ubuntu/
-        follow: yes
+        local_follow: yes
```

Comment:
What does this do?

Reply:
Follows local symlinks. The files in `/etc/letsencrypt/live/multinet.app` are symlinked to `/etc/letsencrypt/archive/multinet.app`. The link names in the live folder never change, but I think the link targets do in the archive folder. This just lets us get the live version of the files in the archive